docs/networking/diagnostics/inspecting-network-connections-with-netstat/index.md

```diff
@@ -77,42 +87,390 @@ unix  2      [ ACC ]     STREAM     LISTENING     24523    public/showq
 unix  2      [ ACC ]     STREAM     LISTENING     24526    private/error
 {{< /output >}}
 
-The first column of the output is the protocol. However, depending on the kind of socket you have, the output will differ. When we are talking about TCP/IP, the `Recv-Q` column shows the number of bytes received by the network interface but not read by the process. Similarly, the `Send-Q` column shows the number of bytes sent to the other side of the TCP/IP connection but not acknowledged so far. The remaining columns of a TCP/IP connection are `Local Address`, `Foreign Address` and `State`.
+The first table displays network connections, and the columns of this table are interpreted as follows:
```

Comment:
Restating explanation of columns as a table, and expanding on some column explanations